### PR TITLE
Lcollins

### PR DESCRIPTION
**Pull request by lcollins11**
Returns a dictionary using the question id as a key and the question title as the value. Working fine as far as I can tell.

### CODE REVIEW

File: src/utils/leetcode_util.py [hunk 1]
```diff
@@ -34,4 +35,48 @@ def api_questions_loadall() -> List[dict]:
...
+            questions = data['data']['recentAcSubmissionList']
+            for question in questions:
+                recent_completions[question['id']] = question['title']
+        else:
+            print(f"Response returned status code : {response.status_code}")
+
+        return recent_completions
+
+
+# print(LeetcodeUtil.get_recent_submissions("King_of_Cheese"))
```
Comment (jacobmillsfl) [inline src/utils/leetcode_util.py]: Remove test in comment

**Review by jacobmillsfl — CHANGES_REQUESTED** [review]
Looks good, remove comment of test and push again

File: src/utils/leetcode_util.py (hunk 1 above)
Comment (lcollins11) [reply src/utils/leetcode_util.py]: Done

File: src/utils/leetcode_util.py [hunk 2]
```diff
@@ -5,6 +5,7 @@ class LeetcodeUtil:
     GRAPH_URL = "https://leetcode.com/graphql/"
     PROBLEM_URL = "https://leetcode.com/problems/"
     ALL_PROBLEMS_URL = "https://leetcode.com/api/problems/all/"
+    data_limit = 100
```
Comment (jacobmillsfl) [inline src/utils/leetcode_util.py]: Use `DATA_LIMIT` as all caps convention for class global variables

Comment (lcollins11) [reply src/utils/leetcode_util.py]: Good spot, thanks!

**Review by jacobmillsfl — APPROVED** [review]
Nice job

**Pull request merged**